src/server/api/controllers/boards.controller.js

```diff
@@ -59,7 +59,15 @@ const getBoardsByCreatorId = async (id) => {
 
   const boards = await knex('users')
     .join('boards', 'users.id', '=', 'boards.creatorId')
-    .select('*')
+    .select(
+      'users.fullName',
+      'boards.id',
+      'boards.creatorId',
+      'boards.createdOn',
+      'boards.deadline',
+      'boards.isDeleted',
```

Comment:
I would actually create a condition so that these boards don't get selected at all.

```js
.where('isDeleted', false)
```

and then you can omit this field.